onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -383,6 +383,13 @@ def _scatter_add_input_wrangler(
     return args, kwargs
 
 
+def _scatter_reduce_input_wrangler(
+    args: list[Any], kwargs: dict[str, Any]
+) -> tuple[list[Any], dict[str, Any]]:
+    kwargs["reduce"] = args.pop(4)  # put the string into kwargs, otherwise FullGraph mode will cannot find get 'reduce' argument
```

Comment:
```suggestion
    # Put the string into kwargs, otherwise FullGraph mode will cannot find get 'reduce' argument
    kwargs["reduce"] = args.pop(4)
```

nit: Prefer a separate line for comments. Also capitalize the first letter